From 7e0a2ea46c258642a7580581ff1fbe2bd5743b40 Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Wed, 25 Nov 2020 19:11:12 +0100
Subject: [PATCH 011/841] feat(dt-bindings): add MCE DT bindings for STM32MP13

This makes the region definition easier in DT, with explicit parameters.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git a5a2e90d86e313de3b9563e9462a2c5125557df7 feat(dt-bindings): add MCE DT bindings for STM32MP13]
Change-Id: I4e3f6e6d31288ccec807f3fc71544d8cf39ef1e5
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
---
 include/dt-bindings/soc/stm32mp13-mce.h | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 include/dt-bindings/soc/stm32mp13-mce.h

diff --git a/include/dt-bindings/soc/stm32mp13-mce.h b/include/dt-bindings/soc/stm32mp13-mce.h
new file mode 100644
index 000000000..15006fc61
--- /dev/null
+++ b/include/dt-bindings/soc/stm32mp13-mce.h
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: GPL-2.0+ OR BSD-3-Clause */
+/*
+ * Copyright (C) 2020, STMicroelectronics - All Rights Reserved
+ */
+
+#ifndef _DT_BINDINGS_STM32MP13_MCE_H
+#define _DT_BINDINGS_STM32MP13_MCE_H
+
+#define MCE_PLAINTEXT  U(0)
+#define MCE_ENCRYPT    U(1)
+
+#endif /* _DT_BINDINGS_STM32MP13_MCE_H */
-- 
2.47.2

